### PR TITLE
MINOR: Use https instead of http in links

### PR DESCRIPTION
**Pull request by ijuma**
Verified that the https links work.

I didn't update the license header in this PR since that touches
so many files. Will file a separate one for that.

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Comment by rajinisivaram** [comment]
@ijuma The builds seem to have failed.

File: doap_Kafka.rdf [hunk 1]
```diff
@@ -13,30 +13,30 @@
...
     <bug-database rdf:resource="https://issues.apache.org/jira/browse/KAFKA" />
-    <mailing-list rdf:resource="http://kafka.apache.org/contact.html" />
-    <download-page rdf:resource="http://kafka.apache.org/downloads.html" />
+    <mailing-list rdf:resource="https://kafka.apache.org/contact.html" />
+    <download-page rdf:resource="https://kafka.apache.org/downloads.html" />
     <programming-language>Scala</programming-language>
-    <category rdf:resource="http://projects.apache.org/category/big-data" />
+    <category rdf:resource="https://projects.apache.org/category/big-data" />
```
Comment (omkreddy) [inline doap_Kafka.rdf]: nit: looks like current url in not working.  New url: https://projects.apache.org/projects.html?category#big-data

**Review by omkreddy — APPROVED** [review]
@ijuma  Thanks for the PR. LGTM. left minor comment. also there is a rat check failure.

**Comment by ijuma** [comment]
No code changes, so test failure was unrelated (one of the builds passed too).

**Pull request merged**